workflow/Snakefile

```diff
@@ -14,8 +14,8 @@ RESULTS_DIR = config["results_dir"]
 
 rule all:
 	input: 
-		expand(os.path.join(RESULTS_DIR, "{dataset}", "NumCandidateEnhGene.txt"), dataset=dataset_config['dataset']),
-		expand(os.path.join(RESULTS_DIR, "{dataset}",  "NumTSSEnhGene.txt"), dataset=dataset_config['dataset']),
+		expand(os.path.join(RESULTS_DIR, "{dataset}", "NumCandidateEnhGene.tsv"), dataset=dataset_config['dataset']),
+		expand(os.path.join(RESULTS_DIR, "{dataset}",  "NumTSSEnhGene.tsv"), dataset=dataset_config['dataset']),
 		expand(os.path.join(RESULTS_DIR, "{dataset}", "NumEnhancersEG5kb.txt"), dataset=dataset_config['dataset']),
 		expand(os.path.join(RESULTS_DIR, "{dataset}", "SumEnhancersEG5kb.txt"), dataset=dataset_config['dataset']),
 		expand(os.path.join(RESULTS_DIR, "{dataset}", "ActivityOnly_features.tsv.gz"), dataset=dataset_config['dataset']),
```

Comment:
Why not change all the file extensions to .tsv so it's consistent?

Reply:
I don't think all of them are actually tsv files. For tsv files, I believe we'd want the first line to be the column names.
```
(encode_e2g_features) [atan5133@sh03-04n24 /oak/stanford/groups/engreitz/Users/atan5133/encode_e2g_features/results/K562_cellranger_1] (job 28846588) $ head -1 NumEnhancersEG10kb.txt
genic|chr10:100006248-100006983 2
```

My approach is to clean things up as I go, as I would overwhelm myself if I try to tackle many clean up issues in bulk. I think it's much better to have some files name properly (as tsv) vs no files named properly

Reply:
Got it, thanks.